panel/widgets/slider.py

```diff
@@ -241,6 +258,20 @@ def _update_value(self, event):
         finally:
             self._syncing = False
 
+    def _update_value_throttled(self, event):
+        values = self.values
+        if self.value_throttled not in values:
+            self.value_throttled = values[0]
+            return
+        index = self.values.index(self.value_throttled)
+        if self._syncing_throttled:
+            return
+        try:
+            self._syncing_throttled = True
+            self._slider.value = index
+        finally:
+            self._syncing_throttled = False
```

Comment:
Can you explain the purpose behind this method?

Reply:
I'm properly not understanding this correct, but the method is almost the same as the method above it `_update_values()` with no-throttled variables changed to throttled variables.

Reply:
Ah, in that case it's my fault for not properly documenting that code :)